README.md

```diff
@@ -1,14 +1,27 @@
 # <img src="https://cloud.githubusercontent.com/assets/80639/7736468/c78ac686-fef8-11e4-9931-cc3ef8fd37a0.png" width="600" alt="The Tessel Project">
 
-<a href="https://tessel-slack.herokuapp.com/">Join the conversation on our community chat:  <img src="https://tessel-slack.herokuapp.com/badge.svg"></a>
+Looking for the entry point to the Tessel Project? You've come to the right place.
```

Comment:
Should we link to t2-start and t1-start here? I feel like linking people to "how to use Tessel" via Github is so hard